industrial_ci/src/tests/source_tests.sh

```diff
@@ -80,10 +98,11 @@ function run_clang_tidy_check {
     ici_parse_env_array clang_tidy_args CLANG_TIDY_ARGS
 
     ici_run "install_clang_tidy" ici_install_pkgs_for_command clang-tidy clang-tidy "$(apt-cache depends --recurse --important clang  | grep "^libclang-common-.*")"
+    test -n "$CLANG_TIDY_BASE_REF" && ici_setup_git_client
```

Comment:
This will fail, if `CLANG_TIDY_BASE_REF` is empty (`set -e`...)
Either `test -z "$CLANG_TIDY_BASE_REF" || ici_setup_git_client` or
```
if [ -n "$CLANG_TIDY_BASE_REF" ]; then
  ici_setup_git_client
fi
```
I prefer the latter, because it is more expressive.

Reply:
I see. Fixed.